### PR TITLE
fixed issue 165

### PR DESCRIPTION
**Pull request by detroxx92**
Bugfix to the issue https://github.com/flagbug/YoutubeExtractor/issues/165

### CODE REVIEW

File: YoutubeExtractor/YoutubeExtractor/DownloadUrlResolver.cs [hunk 1]
```diff
@@ -215,7 +215,12 @@ private static string GetAdaptiveStreamMap(JObject json)
         {
             JToken streamMap = json["args"]["adaptive_fmts"];
 
-            return streamMap.ToString();
+            // bugfix: adaptive_fmts is missing in some videos, use url_encoded_fmt_stream_map instead
```
Comment (flagbug) [inline YoutubeExtractor/YoutubeExtractor/DownloadUrlResolver.cs]: Please fix the formatting to adhere to the same style as the rest of the project

File: YoutubeExtractor/YoutubeExtractor/DownloadUrlResolver.cs [hunk 2]
```diff
@@ -215,6 +215,11 @@ private static string GetAdaptiveStreamMap(JObject json)
         {
             JToken streamMap = json["args"]["adaptive_fmts"];
 
+            // bugfix: adaptive_fmts is missing in some videos, use url_encoded_fmt_stream_map instead
+            if (streamMap == null) {
```
Comment (flagbug) [inline YoutubeExtractor/YoutubeExtractor/DownloadUrlResolver.cs]: @detroxx92  Move the bracket here to the next line

**Comment by detroxx92** [comment]
Welp didn't saw this in the first place :)

**Comment by flagbug** [comment]
Thanks!

**Pull request merged**